### PR TITLE
Merge request from the feature branch Jfb-1 to develop

### PR DESCRIPTION
**Pull request by mamyr**

### CODE REVIEW

File: src/com/andersen/lesson2/Ticket.java [hunk 1]
```diff
@@ -116,4 +116,12 @@ public void save_price(int price){
         this.price = price;
         System.out.println("Price is saved.");
```
Comment (deolexx) [inline src/com/andersen/lesson2/Ticket.java]: As for now it is acceptable but Using System.out.println  for logging is generally not recommended

File: src/com/andersen/lesson2/TicketService.java [hunk 2]
```diff
@@ -10,7 +10,32 @@ public static void main(String[] args) {
...
+        for(int i=0; i<tickets.length; i++){
+            System.out.println("ticket.id["+i+"]="+tickets[i].getID());
+        }
+    }
+
+    public static Ticket getTicketByID(Ticket[] tickets, String ID){
+        int foundIndex = 0;
+        for(int i=0; i<tickets.length; i++){
+            if(tickets[i].getID().equals(ID)){
+                System.out.println("ticket.id["+i+"]="+tickets[i].getID());
```
Comment (deolexx) [inline src/com/andersen/lesson2/TicketService.java:46]: If you need to concatenate multiple strings together it is better to use StringBuilder or at least String.format()

**Review by deolexx — APPROVED** [review]

File: src/com/andersen/lesson2/TicketService.java (hunk 2 above)
Comment (mamyr) [reply src/com/andersen/lesson2/TicketService.java:46]: Forgot. It was long time ago.

File: src/com/andersen/lesson2/Ticket.java (hunk 1 above)
Comment (mamyr) [reply src/com/andersen/lesson2/Ticket.java]: I will fix it

**Pull request merged**